src/app/app.container.tsx

```diff
@@ -1,17 +1,22 @@
 import React, { Component } from 'react';
 import { Route, Switch } from 'react-router-dom';
 import { CounterContainer } from '../containers/counter/counter.container';
+import { DetailContainer } from '../containers/detail/detail.container';
 import { HomepageContainer } from '../containers/homepage/homepage.container';
+import { AdminLayoutContainer } from '../containers/sharedlayouts/admin-layout.container';
 import './app.container.less';
 
 export class AppContainer extends Component {
     render() {
         return (
-            <Switch>
-                <Route exact path="/" component={HomepageContainer} />
-                <Route path="/counter" component={CounterContainer} />
-                <Route render={() => <div>No routes here</div>} />
-            </Switch>
+            <AdminLayoutContainer>
```

Comment:
@kfrancois is this the way to go?

Reply:
Sure, best to think about where we want to show which environment though. I think `admin` isn't appropriate since we're just working locally with 1 user

Reply:
I was thinking in user stories. Admin for making / editing of surveys and viewing statistics. User for filling in said surveys (and viewing more limited results).